test-cases/longevity/longevity-100gb-4h.yaml

```diff
@@ -19,3 +19,5 @@ user_prefix: 'longevity-100gb-4h'
 space_node_threshold: 64424
 
 use_mgmt: true
+pre_create_schema: True
+sstable_size: 100
```

Comment:
please also configure the correct `compaction_strategy` if the `sstable_size` goes hand by hand with it

Reply:
IIUC, it'll set the sstable size to be 100mb.
Did you test the effect of it?

Reply:
> please also configure the correct `compaction_strategy` if the `sstable_size` goes hand by hand with it

@fruch,  why overriding the default value with the default value itself?
BTW, if in the future, scylla will change the default from STCS to ICS this file will not have to be changed.

Reply:
> IIUC, it'll set the sstable size to be 100mb.
> Did you test the effect of it?

it was tested on smaller sized on Dtest. i will run a test on this fork soon.

Reply:
test is running, see comment below